src/lib/tls/tls13_pqc/hybrid_public_key.h

```diff
@@ -0,0 +1,111 @@
+/**
+* Composite key pair that exposes the Public/Private key API but combines
+* multiple key agreement schemes into a hybrid algorithm.
+*
+* (C) 2023 Jack Lloyd
+*     2023 Fabian Albert, René Meusel - Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_TLS_13_HYBRID_KEM_PUBLIC_KEY_H_
+#define BOTAN_TLS_13_HYBRID_KEM_PUBLIC_KEY_H_
+
+#include <botan/pubkey.h>
+
+#include <botan/tls_algos.h>
+
+#include <memory>
+#include <vector>
```

Comment:
They could. Though, actually, I got used to separating STL from botan includes like that on purpose. For better visual separation. 😅

Should I stop doing that?

Reply:
I think it's fine as long as you follow the same ordering that `clang-format` would otherwise based on `IncludeCategories` configuration, eg first botan headers then std headers. (which you do here)